lumicks/pylake/detail/image.py

```diff
@@ -3,6 +3,37 @@
 import numpy as np
 
 
+class InfowaveCode(enum.IntEnum):
+    discard = 0  # this data sample does not contain useful information
```

Comment:
I'd be in favor of having the naming here be consistent with Bluelake. Renaming the one in Bluelake is also fine with me, though, if that makes more sense.

Reply:
I'm not a fan of `combine` and `store` naming in Bluelake because it's very suggestive of serial data processing. I'll make a PR there to make it consistent.